devops/openshift/test/configmap-envvar.properties

```diff
@@ -45,4 +45,7 @@ MAIL_HOST=
 DSS_SERVICE_METADATA_RESOURCE=file://${application.dir}/config/dss-metadata.xml
 DSS_SERVICE_METADATA_ENTITYID=https://esign.v2.st.signatureservice.se/signservice-frontend/metadata/4321a222112
 DSS_SERVICE_ACTION_URL=https://esign.v2.st.signatureservice.se/signservice-frontend/request/4321a222112
-DSS_SERVICE_IDPURL=https://idp-oob.systemtest.ineradev.org:443/saml
\ No newline at end of file
+DSS_SERVICE_IDPURL=https://idp-oob.systemtest.ineradev.org:443/saml
+KUNDPORTALEN_BASE_URL=http://localhost:8020
```

Comment:
I think you have to change this to ${webcert.host.url} as a value. Because the integration tests will not find the stub otherwise.